packages/flight-icons/.env.template

```diff
@@ -1,7 +0,0 @@
-# This is an api key created from the account of [your name here]
```

Comment:
@jorytindall this file should not be removed

Reply:
Thanks for catching that, I failed to duplicate it and just renamed it.